### ISSUE TITLE
Travis PR build failed starting 2868

### ISSUE THREAD

**Issue by xinzweb** [greenplum-db/gpdb#2885]
Please look at the failure staring https://travis-ci.org/greenplum-db/gpdb/builds/260071592

The failure is:

```
$ wget http://ftp.jaist.ac.jp/pub/apache/apr/${APR_TAR}
--2017-08-02 06:45:44--  http://ftp.jaist.ac.jp/pub/apache/apr/apr-1.5.2.tar.gz
Resolving ftp.jaist.ac.jp (ftp.jaist.ac.jp)... 150.65.7.130, 2001:df0:2ed:feed::feed
Connecting to ftp.jaist.ac.jp (ftp.jaist.ac.jp)|150.65.7.130|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-08-02 06:45:45 ERROR 404: Not Found.
The command "wget http://ftp.jaist.ac.jp/pub/apache/apr/${APR_TAR}" failed and exited with 8 during .
```

Anybody is looking at the ftp failure? Is the website down? @edespino do you have any additional info?

**Comment by Chibin** [comment]
> Anybody is looking at the ftp failure? Is the website down? @edespino do you have any additional info?

Just took a quick look, http://ftp.jaist.ac.jp/pub/apache/apr/
They rev'ed the apr and apr-util versions.
apr-1.6.2
apr-util-1.6.0

**Comment by cjcjameson** [comment]
addressed with #2888
